script/export_for_foi/export_mappings.sql

```diff
@@ -0,0 +1,7 @@
+COPY (
+  SELECT sites.abbr, path, new_url, suggested_url, archive_url,
```

Comment:
I think we should `archive_url AS custom_archive_url` here, and then explain in the accompanying text how default archive URLs are constructed, with a [link to Bouncer doing that](https://github.com/alphagov/bouncer/blob/568ece01cefe536d5439fa39fe6df4a403539e6e/lib/bouncer/request_context.rb#L51). That means we can be more explicit about which hostname is used and that the request path and querystring are used, and avoids inflating the data inaccurately.